### DISCUSSION TITLE
Fundamentos SQL Server- Introdução: Qual Docker devo baixar?

### DISCUSSION THREAD

**Discussion by oimarcella** [balta-io/forum#422] [Tira Dúvidas] [answered]
Uso windows 11, tenho WSL instalado, mas qual versão do Docker eu devo instalar já que o WSL é um subsistema Linux. Baixo para windows ou linux?

**Comment by andrebaltieri** [comment, marked as answer]
Fica a seu critério @mlasa! Se você usar o do Windows, ele vai instalar no WSL também!
Aqui tem as configurações que uso se quiser deixar igual do curso:
https://balta.io/blog/sql-server-docker
